src/forms/PublishOptions.jsx

```diff
@@ -0,0 +1,266 @@
+
+  constructor(props) {
+    super(props);
+    this.state = { publishModalOpened: false, formStatus: 'closed', activeTab: 0 };
+  }
+
+  togglePublishModal() {
+    this.setState({ publishModalOpened: !this.state.publishModalOpened });
+  }
+
+  onFormStatusChange(e) {
+    let status = e.target.value;
+    this.setState({ formStatus: status });
+    this.props.dispatch(updateForm({status}));
+    this.props.dispatch(updateFormSettings({ isActive: status === 'open' }));
+  }
+
+  onInactiveMessageChange(e) {
+    this.props.dispatch(updateFormSettings({ inactiveMessage: e.target.value }));
+  }
+
+  getFormStatusSection() {
+    const { forms, activeForm, onSaveClick, isForm } = this.props;
+    const form = activeForm ? forms[activeForm] : forms.form;
+    return (
+      isForm && form
+      ? <div>
+          <h4 style={ styles.dialogSubTitle }>Set form status</h4>
+          <RadioGroup
+            onChange={ this.onFormStatusChange.bind(this) }
+            childContainer="div"
+            value={ form.status }
+            name="formStatusRadio">
+            <Radio value="open" ripple>Live, accepting submissions</Radio>
+            <Radio value="closed">Closed, not accepting submissions</Radio>
+          </RadioGroup>
+          {
+            this.state.formStatus && this.state.formStatus == 'closed'
+            ? <div>
+              <Textfield
+                onChange={this.onInactiveMessageChange.bind(this)}
+                label="We are not currently accepting submissions..."
+                value={form.settings.inactiveMessage}
+                style={{ width: '95%', marginLeft: '5%' }}
+              />
+              </div>
+            : null
+          }
+          <div style={ styles.rightAlignButtons }>
+            <Button
+              raised ripple
+              style={{ marginTop: 20 }}
+              onClick={onSaveClick}>
+              { forms.savingForm ? <Spinner/> : null } Apply
+            </Button>
+          </div>
+        </div>
+      : null
+    );
+  }
+
+  render() {
+
+    const { onOpenPreview, forms, activeForm, onSaveClick, iframeCode, scriptCode, standaloneCode } = this.props;
+
+    return (
+      <div style={styles.leftPan}>
+
+          <div>
+            <Button
+              raised ripple primary
+              onClick={onOpenPreview}
+              style={{ width: 150, marginRight: 10, backgroundColor: '#4565A1' }}>
+              Preview
+            </Button>
+            <Button
+              style={{ width: 150, backgroundColor: '#358D66' }}
+              raised ripple colored
+              onClick={onSaveClick}>
+              { forms.savingForm || forms.loadingGallery ? <Spinner/> : null } Save
+            </Button>
+
+            {
+              activeForm
+              ? <Button
+                  style={{ width: 310, marginTop: 10, backgroundColor: '#353B43' }}
+                  raised ripple accent
+                  onClick={this.togglePublishModal.bind(this)}>
```

Comment:
Before merging if you can bind this in the constructor i'd be great. Otherwise this could go in another issue or refactor :) 

@pablocubico